services/openshiftbuilddeploy/src/index.js

```diff
@@ -250,6 +256,13 @@ const messageConsumer = async msg => {
       buildconfig.spec.strategy.customStrategy.env.push({"name": "PROMOTION_SOURCE_ENVIRONMENT","value": promoteSourceEnvironment})
       buildconfig.spec.strategy.customStrategy.env.push({"name": "PROMOTION_SOURCE_OPENSHIFT_PROJECT","value": openshiftPromoteSourceProject})
     }
+    if (!R.isEmpty(projectOpenShift.groups)) {
+      projectOpenShift.groups.forEach(group => {
+        if (!R.isEmpty(group.envVariables)){
+          buildconfig.spec.strategy.customStrategy.env.push({"name": "LAGOON_GROUP_VARIABLES", "value": JSON.stringify(group.envVariables)})
```

Comment:
If there were two groups with env vars for a project, then wouldn't this loop produce duplicates environment names in the build config? How does openshift handle that, does the last one win or do they get merged?

Reply:
yes I think the last one wins

Reply:
correct